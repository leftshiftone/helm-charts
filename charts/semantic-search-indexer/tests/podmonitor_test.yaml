suite: test podmonitor
templates:
  - podmonitor.yaml
tests:
  - it: default
    set:
      podMonitor.enabled: true
    asserts:
      - isKind:
          of: PodMonitor
      - equal:
          path: metadata.name
          value: RELEASE-NAME-semantic-search-indexer
      - matchSnapshot:
          path: metadata
      - matchSnapshot:
          path: spec
