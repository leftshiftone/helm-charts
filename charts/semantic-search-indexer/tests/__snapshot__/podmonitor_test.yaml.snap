default:
  1: |
    labels:
      app.kubernetes.io/instance: RELEASE-NAME
      app.kubernetes.io/managed-by: Helm
      app.kubernetes.io/name: semantic-search-indexer
      app.kubernetes.io/version: 0.0.0
      helm.sh/chart: semantic-search-indexer-0.0.0
      release: RELEASE-NAME
    name: RELEASE-NAME-semantic-search-indexer
    namespace: NAMESPACE
  2: |
    namespaceSelector:
      matchNames:
      - NAMESPACE
    podMetricsEndpoints:
    - interval: 10s
      path: /metrics
      port: http
    podTargetLabels:
    - application
    - app_kubernetes_io_instance
    - app_kubernetes_io_name
    - app_kubernetes_io_version
    selector:
      matchLabels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/name: semantic-search-indexer
